jest-puppeteer.config.js

```diff
@@ -0,0 +1,20 @@
+const config = require( '@wordpress/scripts/config/puppeteer.config.js' );
+
+// https://github.com/smooth-code/jest-puppeteer/blob/master/packages/jest-environment-puppeteer/README.md#jest-puppeteerconfigjs
+module.exports = {
+	launch: {
+		...( config.launch || {} ),
+		args: [
+			// https://peter.sh/experiments/chromium-command-line-switches/
+			'--disable-gpu',
+			'--no-sandbox',
+			'--lang=en-US',
+			'--disable-setuid-sandbox',
+			'--disable-dev-shm-usage',
```

Comment:
Are all of these arguments necessary? I know `--disable-dev-shm-usage` was the main one to add here but if the others aren't needed, let's not add them unless there's a good reason to.

Reply:
Updated.